src/server/common/dbconn.cpp

```diff
@@ -586,17 +586,17 @@ bool dbconn::do_any_authors_exist(const std::string& instance_version, const std
 //
 // handle complex query results
 //
-void dbconn::get_complex_results(mariadb::connection_ref connection, rs_base& base, const std::string& sql, const sql_parameters& params)
+void dbconn::get_complex_results(const mariadb::connection_ref& connection, rs_base& base, const std::string& sql, const sql_parameters& params)
 {
-	mariadb::result_set_ref rslt = select(connection, sql, params);
+	mariadb::result_set_ref rslt = select(std::move(connection), sql, params);
```

Comment:
This seems wrong. You can't move from a const reference, and I noticed this same change was applied several times to this file. @jj-marr Is this false linter positive?

Reply:
It is wrong. Sometimes the linter gives bad suggestions in this case. I believe you should be able to just mark `const mariadb::connection_ref& connection` and the linter should let it through without the `std::move`.